sdk/rust-examples/aesctr-native/Makefile

```diff
@@ -0,0 +1,26 @@
+# Example Makefile
```

Comment:
Do we even need a `Makefile` for each Rust example?

Reply:
Probably not, but each example currently has one.  This can be fixed in another PR I think.